### DISCUSSION TITLE
scrolling issues in nextjs

### DISCUSSION THREAD

**Discussion by usmandevops123** [vercel/next.js#44217] [Help]
why the page is displaying from the bottom after routing in nextjs? 

This is my code when it will redirect the page will come from the bottom side first. How can I control it?
my code is 
`   <button type="submit" onClick={() => router.push("/blogs")} className="bg-white btn btn-register text-black">
                                {t('find')}
                            </button> `

**Comment by ciruz** [comment]
Hey @usmandevops123 try this:

```javascript
router.push("/blogs").then(() => window.scrollTo(0, 0));
```

**Reply by usmandevops123** [reply]
Thanks
